broker-core/src/main/java/org/apache/qpid/server/logging/messages/UserResourceLimit_logmessages.properties

```diff
@@ -0,0 +1,23 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# User Resource Limit logging message i18n strings.
+ACCEPTED = URL-1001 : Accepted : {0} {1} by {2} : {3}
+REJECTED = URL-1002 : Rejected : {0} {1} by {2} : {3}
+DEFERRED = URL-1003 : Deferred : {0} {1} by {2}
```

Comment:
ACCEPTED can not be deleted, we need to log the accepted connection as well with the current connection counter value. Rejecting a new connection because of the breaking the limits is not a regular close operation.